taxonomies/categories.txt

```diff
@@ -84017,6 +84073,9 @@ fr:Boudins blancs
 agribalyse_proxy_food_code:en:8803
 wikidata:en:Q2751264
 
+<fr:Boudins blancs
+fr:Boudins blancs aromatisés
```

Comment:
```suggestion
en:Flavored boudins blancs
fr:Boudins blancs aromatisés
```

https://www.wikidata.org/wiki/Q2751264

Reply:
Hello @benbenben2 the whole 'boudin blanc' hierarchy is in French: it is bizarre to add an english tags only here. In addition, I don't think that 'boudin blanc' translates to 'boudin blanc' in English. Maybe stg like 'white sausage'. I prefer to leave it as it is, for someone more knowledgeable than me in English to add 'en:' to the whole hierarchy, if that's ok with you.